test/models/DateOfChangeSpec.scala

```diff
@@ -52,6 +52,49 @@ class DateOfChangeSpec extends PlaySpec {
 
     }
 
+    "fail form validation when given a future date in pre-app" in {
+      val model =    Map (
+        "dateOfChange.day" -> Seq("24"),
+        "dateOfChange.month" -> Seq("2"),
+        "dateOfChange.year" -> Seq(LocalDate.now().plusYears(1).getYear.toString)
+      )
+
+      DateOfChange.formRulePreApp.validate(model) must be(
+        Invalid(
+          Seq(
+            Path \ "dateOfChange" -> Seq(ValidationError("error.required.dateofchange.future")))
+        ))
+
+    }
+
+    "fail form validation when given a past 1900 date in pre-app" in {
+      val model =    Map (
+        "dateOfChange.day" -> Seq("24"),
+        "dateOfChange.month" -> Seq("2"),
+        "dateOfChange.year" -> Seq("1800")
+      )
+
+      DateOfChange.formRulePreApp.validate(model) must be(
+        Invalid(
+          Seq(
+            Path \ "dateOfChange" -> Seq(ValidationError("error.required.dateofchange.1900")))
+        ))
+    }
+
+    "fail form validation when missing day field in pre-app" in {
```

Comment:
Suggestion; We should cover all missing day/month/year combos here.

Reply:
Done